lib/Remote/RemoteWebElement.php

```diff
@@ -403,6 +403,27 @@ public function equals(WebDriverElement $other)
         ]);
     }
 
+    /**
+     * Take screenshot of a specific element.
+     *
+     * @param string $save_as The path of the screenshot to be saved.
+     * @return string The screenshot in PNG format.
+     */
+    public function takeElementScreenshot($save_as = null)
```

Comment:
camelCale $saveAs

Reply:
We are now mixing camelCase and underscore_case in the codebase (will be united to camelCase in 2.0), so anything is now acceptable :)

Reply:
I kept the existing code.